### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sandstrom**
Missing comma

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ ENV.webhooks = {
         return {
           apiKey: <your-api-key>
         };
-      }
-      didActivate: true
+      },
+      didActivate: true,
```
Comment (marcoow) [inline README.md]: extra comma

Comment (sandstrom) [reply README.md]: hi @marcoow! I don't understand what you're trying to say

Comment (marcoow) [reply README.md]: I don't think we use trailing commas in this repo

Comment (sandstrom) [reply README.md]: A trailing comma in the readme doesn't seem like the worst of sins. It's removed now anyways.

**Pull request merged**